apps/web/src/pages/brand/tabs/LayoutsListPage.tsx

```diff
@@ -71,52 +72,54 @@ export function LayoutsListPage({ handleLayoutAnalytics }: LayoutsListPageProps)
     setActiveScreen(ActivePageEnum.LAYOUTS_LIST);
   };
 
-  const editLayout = (id: string) => {
+  const editLayout = (id?: string) => {
+    if (typeof id === 'undefined') return;
+
     setEditId(id);
     setActiveScreen(ActivePageEnum.EDIT_LAYOUT);
     handleLayoutAnalytics('Edit screen opened', { layoutId: id });
   };
 
-  const columns: ColumnWithStrictAccessor<Data>[] = [
+  const columns: IExtendedColumn<ILayoutEntity>[] = [
     {
       accessor: 'name',
       Header: 'Name',
-      Cell: ({ name }: any) => (
-        <Tooltip label={name}>
+      Cell: withCellLoading(({ row: { original } }) => (
```

Comment:
When we do override the `Cell` and want to have a skeleton we would have to wrap the component with the HOC for loading, otherwise we would have to handle the loading state manually, in some cases, it might be useful, for ex. if you want to show the different loading skeletons.